powerauth-admin/src/main/java/io/getlime/security/app/admin/controller/ActivationController.java

```diff
@@ -214,38 +214,29 @@ public String activationDetail(
      *
      * @param applicationId           Application ID of an associated application.
      * @param userId                  User ID.
-     * @param activationOtpValidation Activation OTP validation mode.
+     * @param commitPhase             Specifies when activation is committed.
      * @param activationOtp           Activation OTP code.
      * @param model                   Model with passed parameters.
      * @param redirectAttributes      Redirect attributes.
      * @return Redirect the user to activation detail.
      */
     @GetMapping("/activation/create")
     public String activationCreate(@RequestParam("applicationId") String applicationId, @RequestParam("userId") String userId,
-                                   @RequestParam("activationOtpValidation") String activationOtpValidation,
+                                   @RequestParam("commitPhase") String commitPhase,
```

Comment:
🤔 Should it be `required=false`? Or are we sure it is safe because it is called by our admin GUI?

Reply:
The controller is used only by admin GUI which specifies the parameter.